src/csharp/Microsoft.Spark/Utils/JvmObjectUtils.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Spark.Interop.Ipc;
+using System.Collections.Generic;
+
+namespace Microsoft.Spark.Utils
+{
+    /// <summary>
+    /// JvmObjectUtils is used to hold basic general helper functions that
+    /// are used within ML scope.
+    /// </summary>
+    internal class JvmObjectUtils
+    {
+        /// <summary>
+        /// Helper function for constructing the mapping between java class name and dotnet class type.
```

Comment:
Add description what the method does, ie searches assemblies and how it uses the parentType and javaClassFields to construct the dotnet class type mapping.